examples/markdown_challenges.md

```diff
@@ -52,6 +52,125 @@ an HTML href tag that uses `http_base`.
 - Organization: ACI
 - Created: 2020-06-24
 
+## Network Options
+
+This optional section can be used to configure the Docker network associated
+with each instance of this challenge. The available options are listed below,
+along with example usage.
+
+The `Internal` option disables Internet access, restricting network traffic to
+the instance's containers. This is equivalent to passing the [`--internal`](https://docs.docker.com/engine/reference/commandline/network_create/#network-internal-mode) flag to [`docker network create`](https://docs.docker.com/engine/reference/commandline/network_create/). Specify a boolean value, as shown below. Defaults to `false`.
+
+- Internal: true
```

Comment:
Aside from convenience on the backend, are there other reasons to separate "Network Options" out from "Container Options" below in the author-facing configuration files?

Reply:
I think there is, only because of the multi-container challenge support. If `problem.md` has sections like:

```markdown
## Container Options: work

- Memory: 1G

## Container Options: randomDnsName

- Memory: 256M
```

then it's ambiguous which should have a hypothetical `InternalNetwork` key, or what would happen if multiple were defined with different values, etc.

It admittedly feels a little extraneous to have a separate section for one option, although in the future, it might eventually become helpful to expose additional network-related options (IPv6? Maybe Docker will eventually add built-in [`tc` rule support](https://github.com/moby/moby/issues/26767)?)

Reply:
Oh I just saw your proposed `overrides` key. In that case, I think it probably would be fine to combine the two sections.

Reply:
I've got a good idea how to easily catch the non-overridable aspect of network configuration options as parse time errors.  I think as long as we call out in the documentation that network options in general (in case we add more) are non-overridable, it won't feel hacky.